include/zephyr/bluetooth/audio/csip.h

```diff
@@ -237,6 +238,70 @@ int bt_csip_set_member_sirk(struct bt_csip_set_member_svc_inst *svc_inst,
 int bt_csip_set_member_get_sirk(struct bt_csip_set_member_svc_inst *svc_inst,
```

Comment:
The new _get_info() gets the SIRK (and other info). Is the _get_sirk() then needed anymore? Should it be deprecated and removed?

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/86991 exists for that :)